docs/api/notification.md

```diff
@@ -97,12 +145,57 @@ Returns:
 
 Emitted when the user clicks the "Reply" button on a notification with `hasReply: true`.
 
-#### Event: 'action' _macOS_
+```js
+const { Notification, app } = require('electron')
+
+app.whenReady().then(() => {
+  const n = new Notification({
+    title: 'Send a Message',
+    body: 'Body Text',
+    hasReply: true,
+    replyPlaceholder: 'Message text...'
+  })
+
+  n.on('reply', (e, reply) => console.log(`User replied: ${reply}`))
+  n.on('click', () => console.log('Notification clicked'))
+
+  n.show()
+})
+```
+
+#### Event: 'action' _macOS_ _Windows_
 
 Returns:
 
 * `event` Event
-* `index` number - The index of the action that was activated.
+* `actionIndex` number - The index of the action that was activated.
+* `selectionIndex` number _Windows_ - The index of the selected item, if one was chosen. -1 if none was chosen.
```

Comment:
Can we move these to properties on the `event` object? That way we can easily extend it to other actions in the future.

Also, would it be possible to add a `type` property to the event as to know which action triggered the event?

Reply:
@samuelmaddock that'd make it become semver-major. Though i suppose I can just backport a semver-minor compatible version that leaves `index` alone?